### ISSUE TITLE
Multiplication with units behaves strangely

### ISSUE THREAD

**Issue by benesch** [sass/libsass#951]
In the latest libsass, this fails

```sass
$ratio: (5em / 15px);

.out {
    padding: $ratio  * 30px;
}
```
with `test.scss:4: 10em/px*px is not a valid CSS value`. Works in older versions (roughly pre-3.1.0, but still investigating) and Ruby Sass. Curiously, this is still fine:

```sass
.out {
    padding: (5em / 15px)  * 30px;
}
```

May be related to #948, but seemed to warrant its own issue.

**Comment by benesch** [comment]
Whoop, looks like this is fixed in master. Sorry for the noise.

**Comment by benesch** [comment]
For posterity, looks to have been fixed by #797.
